examples/next/pages/ui/components/attribute-management/SimpleProfilePage.tsx

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
+import { useUserAttributes } from '@aws-amplify/ui-react-core';
+import {
+  Button,
+  View,
+  Flex,
+  Heading,
+  TextField,
+  Divider,
+} from '@aws-amplify/ui-react';
+import { UserAttributeKey } from 'aws-amplify/auth';
+
+export const SimpleProfilePage = () => {
+  const [state, handleAttributes] = useUserAttributes();
+  const [isEditMode, setIsEditMode] = React.useState(false);
+  const formRef = React.useRef<HTMLFormElement>(null);
```

Comment:
A `ref` should be a unique identifier and shouldn't be reused for diff components. It also seems unnecessary here so can probably be removed unless it's being used for something I missed